### PR TITLE
CmdSender display range, set default state value

### LINKED ISSUE #370: Command Sender display param ranges
Command Sender currently does not have the ability to view the valid command ranges for a command which makes it hard to know the valid command values

### PR DESCRIPTION
**Pull request by jmthomas**
closes #370

### CODE REVIEW

**Comment by jmthomas** [comment]
Playwright failures due to unmerged PRs. I'll merge master once those are merged and re-run.

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdsender/src/tools/CommandSender/CommandSender.vue
Comment (ryanmelt) [inline openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdsender/src/tools/CommandSender/CommandSender.vue:507]: Were these all unused?

**Review by ryanmelt — APPROVED** [review]

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdsender/src/tools/CommandSender/CommandSender.vue
Comment (jmthomas) [reply openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdsender/src/tools/CommandSender/CommandSender.vue:507]: Sort of. They're unused here but used in the parameter editor. I think Vue just populates what you give it so there was no need to pass them here.

**Pull request merged**
